coursework/product-design/index.md

```diff
@@ -340,36 +356,46 @@ A landing page cannot effectively convert visitors into leads/users if there are
+* Make promotional videos. View past CS3216 final project videos [here](https://www.youtube.com/playlist?list=PL-0mdQVieeceaFm__nZYSQYzhDGF0C9Jg).
+* [Griddit](https://www.youtube.com/watch?v=Jo_2aYXixqs&index=2&t=0s&list=PL-0mdQVieeceaFm__nZYSQYzhDGF0C9Jg), a live chat room app, got Prof Ben Leong to do an AMA using their service.
+* [Letterbox](https://www.youtube.com/watch?v=7TnnLeSPnhA&index=6&t=0s&list=PL-0mdQVieeceaFm__nZYSQYzhDGF0C9Jg) had a member going around the school on an Airwheel and giving out flyers.
 
 DO NOT:
 
-- Paste unremovable stickers on the benches of School of Computing.
-- Put up posters in inappropriate places like restrooms.
+* Paste unremovable stickers on the benches of the School of Computing.
+* Put up posters in inappropriate places like restrooms.
 
 Videos are not the only way to market an app - to obtain marks in the coveted 30%, think out of the box for non-disruptive, creative and effective marketing strategies. We want a well thought-out marketing plan including details on how you would implement it. Your strategy must obviously be realistic, i.e. do not suggest giving out $10 to each user who downloads your app if you do not have a good plan of getting those funds or a millionaire on your team.
 
 <div class="box">
   <strong class="milestone-counter">Milestone 11:</strong> Design a small marketing campaign for your app. Explain which marketing channels you would use and how you would use them to get the word out about your app to attract more users. Provide implementation details (e.g. if you decide to make a promotional video, write out a brief outline of the video contents). Explain why you think your marketing strategy would be effective in gathering users.
 </div>
 
+## Final Assignment Writeup
+
+Your submission will include a write-up that describes how your group has met all the milestones for this assignment:
+
+* It should contain answers to Milestones 1-4 and 10-11.
+* It should also sumamarise your Milestones 5-9. More specifically, it should:
+  * Make it clear how the designs are organised. Which ones are wireframes, mockups or prototypes?
+  * Briefly share the results and observations from your user testing results.
+  * Highlight the main user flows presented in each draft / iteration submitted.
+
 ## Grading Scheme
 
-In view that this is a brand new assignment, the detailed grading scheme is still being worked out by the teaching staff. We ask for your kind understanding. It is possible that the scheme would evolve as we grade, but not to worry, this assignment is only worth 10% of the final grade and the goal is learn the Product Design process, so don't fret over the grading.
+The grading of the assignment is divided into two components: satisfying the compulsory milestones and coolness factor (30%). Excluding Milestone 0, there are **11 milestones** in total, along with an optional Milestone 8b. Milestone 6 is worth 15% while Milestone 9 is worth 10%. The rest are worth 5% each.
```

Comment:
Are we aligned on this?

Reply:
It's kind of implied but I think it would be good to add "(70%)" next to "compulsory milestones" for completeness.

The rest looks fine to me.